OptimoveSDK/Sources/Classes/Optimobile/OptimoveInApp.swift

```diff
@@ -88,15 +94,15 @@ public enum OptimoveInApp {
         Optimobile.sharedInstance.inAppManager.updateUserConsent(consentGiven: consentGiven)
     }
 
-    public static func setDisplayMode(mode: InAppDisplayMode) {
+    static func setDisplayMode(mode: InAppDisplayMode) {
         Optimobile.sharedInstance.inAppManager.presenter.setDisplayMode(mode)
     }
 
-    public static func getDisplayMode() -> InAppDisplayMode {
+    static func getDisplayMode() -> InAppDisplayMode {
         return Optimobile.sharedInstance.inAppManager.presenter.getDisplayMode()
     }
 
-    public static func getInboxItems() -> [InAppInboxItem] {
+    static func getInboxItems(storage: OptimoveStorage) -> [InAppInboxItem] {
```

Comment:
Seems like this is a breaking change. Can we leave the method with no arguments and retrieve the storage inside?

Reply:
Changed it to public with private method that passes storage